Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
It is only really Windows that is difficult to support for 32-bit.  Mac at
moment does build and run in both 32 and 64 bit due to universal binaries.
Linux could work in 32-bit but it uses separate binaries and the 32-bit
market segment is so small.  While MonoMac is still a thing, we need to
keep it usable.